exponent-view-template/ios/exponent-view-template/AppDelegate.m

```diff
@@ -3,10 +3,7 @@
 #import "AppDelegate.h"
 #import "ExpoKit.h"
 #import "EXViewController.h"
-
-#if __has_include(<EXAuth/EXAuth.h>)
-#import <EXAuth/EXAuth.h>
-#endif
+#import <EXAppAuth/EXAppAuth.h>
```

Comment:
curious why you got rid of the `#if __has_include(`s here -- does that mean that ExpoKit apps that exclude the AppAuth module will need to modify this file as well?

Reply:
Well it's going to be a part of Expo. I think it just depends on if we want the users to leave this code here even after removing the module. 
I'm fine either way, but I changed the name of the library `EXAuth` to `EXAppAuth` and it was tricky to track this error because the macro blocks it

Reply:
I think it's worth making it as easy as possible for people to remove modules -- it's not so good if people start thinking they have to go modify a bunch of different/unpredictable files depending on the module.